python/running.md

```diff
@@ -33,7 +25,7 @@ in these lessons. In general, and out in the real world, Python 3 is preferred
 as it receives the most focus from the people who make Python. If you’re 
 starting a new project, and don’t have to use LHCb software, consider trying to 
 use Python 3. You can install both Python 2 and Python 3 using 
-[Anaconda][anaconda].
+[Anaconda](https://www.anaconda.com/download/).
```

Comment:
I guess you don't need to change this as you've added the links at the bottom of the file (thanks!).

Reply:
It doesn't work due as it is inside a callout block so the markdown is rendered by the panels plugin and not gitbook. I can't find the issue on the gitbook repo, but I think you had commented on it?

Reply:
Ah right, yes, GitbookIO/gitbook#1468.